src/Microsoft.DotNet.Interactive/Events/ValueDescriptor.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    public class ValueDescriptor
+    {
+        public ValueDescriptor(string name, Type clrValueType = null)
+        {
+            Name = name;
+            ClrValueType = clrValueType;
+        }
+
+        public string Name { get; }
+
+        [JsonIgnore]
+        public Type ClrValueType { get; }
```

Comment:
```suggestion
        public Type ClrType { get; }
```

Reply:
Or even just `Type`. It's not being serialized so it's pretty obvious that this refers to a CLR type.

Reply:
done